integr8ly/integreatly-operator
# Intly 2746 webapp manifest

create manifest configmap for solution explorer

## Verification
- Create test namespace
- Create installation CR (selfSignedCerts: true)
- Run this code in that namespace
- Once solution explorer goes to phase complete see the completed products in the installation CR containing the host and version values.
